test/rules/EMatcherRule.py

```diff
@@ -3,8 +3,9 @@
 
 class EMatcherRule(AnsibleLintRule):
     id = 'TEST0001'
-    description = 'This is a test rule that looks for lines ' + \
-                  'containing the letter e'
+    description = (
+        'This is a test rule that looks for lines ' + 'containing the letter e'
```

Comment:
This is a good example of why autoformatters are terrible.

Reply:
Congrats! You found a bug in hundreads of lines, one very easy to fix. 

Note. I do not expect this PR to get merged but I hope to use it to learn about things we could imporve and make specific changes.

If it look overall I see a huge number of good changes, some debatable ones and one of two where I do not like the outcome.